src/Chainweb/Pact/SPV.hs

```diff
@@ -71,20 +71,20 @@ pactSPV cdbv l = SPVSupport $ \s o -> readMVar cdbv >>= go s o
     -- extract spv resources from pact object
     go s o cdb = case s of
       "TXOUT" -> txOutputProofOf o >>= \case
-          Left e -> spvError e
+          Left e -> return $ Left e
           Right t -> verifyTransactionOutputProof cdb t
             >>= extractOutputs
-      "TXIN" -> spvError "TXIN is currently unsupported"
-      x -> spvError
+      "TXIN" -> return $ Left "TXIN is currently unsupported"
+      x -> return . Left
         $ "TXIN or TXOUT must be specified to generate valid spv proofs: "
         <> x
 
     txOutputProofOf :: Object Name -> IO (Either Text (TransactionOutputProof SHA512t_256))
     txOutputProofOf o =
       case toPactValue $ TObject o def of
-        Left e -> spvError e
+        Left e -> return $ Left e
         Right t -> case fromJSON . toJSON $ t of
```

Comment:
This is `traverse`.

Reply:
well shoot. Don't tell anyone i missed that 🙏

Reply:
I'm phoning Rob Norris as we speak.